### ISSUE TITLE
For cmeas.cpp: clang warns that abs is given long but has parameter int.

### ISSUE THREAD

**Issue by JoostHuizinga** [sferes2/sferes2#44]
When compiling spheres on OS X, clang prints the following warning when compiling cmeas:

```
../sferes/ea/cmaes.cpp:2273:29: warning: absolute value function 'abs' given an argument of type 'long' but has parameter of type 'int' which may cause truncation of value [-Wabsolute-value]
    inseed = (long unsigned)abs((long)(100*time(NULL)+clock()));
                           ^
../sferes/ea/cmaes.cpp:2273:29: note: use function 'std::abs' instead
    inseed = (long unsigned)abs((long)(100*time(NULL)+clock()));
                            ^~~
                            std::abs
../sferes/ea/cmaes.cpp:2273:29: note: include the header <cstdlib> or explicitly provide a declaration for 'std::abs'
../sferes/ea/cmaes.cpp:2659:29: warning: absolute value function 'abs' given an argument of type 'long' but has parameter of type 'int' which may cause truncation of value [-Wabsolute-value]
    t->seed = (unsigned int)abs((long)(100*time(NULL)+clock()));
                            ^
../sferes/ea/cmaes.cpp:2659:29: note: use function 'std::abs' instead
    t->seed = (unsigned int)abs((long)(100*time(NULL)+clock()));
                            ^~~
                            std::abs
../sferes/ea/cmaes.cpp:2659:29: note: include the header <cstdlib> or explicitly provide a declaration for 'std::abs'
2 warnings generated.
```

I'm compiling with the the following version of clang:
Apple LLVM version 8.0.0 (clang-800.0.38)
Target: x86_64-apple-darwin15.6.0
Thread model: posix

This issue is not sferes breaking, but it does mean the wrong implementation of abs is used.

Replacing abs by std::abs, as suggested by clang, does not work because std is not defined within this scope:

```
../sferes/ea/cmaes.cpp:2273:29: error: use of undeclared identifier 'std'
    inseed = (long unsigned)std::abs((long)(100*time(NULL)+clock()));
```

I am suggesting replacing the two instances with:

`inseed = (long unsigned)labs((long)(100*time(NULL)+clock()));`

Explicitly calling the abs value function for long integers that is defined in stdlib.h.

**Comment by costashatz** [comment]
Sounds like a good idea.. You can do it in a PR.. Thanks..

**Comment by costashatz** [comment]
Fixed in #57
